ui.py

```diff
@@ -1,75 +1,89 @@
 import textwrap
+from functools import partial
 
 import mdpopups
 import sublime
-from LSP.plugin.core.typing import List, Tuple
+from LSP.plugin.core.typing import List, Optional
 
 from .types import CopilotPayloadCompletion
-from .utils import get_copilot_view_setting, reformat, set_copilot_view_setting
+from .utils import clamp, get_copilot_view_setting, reformat, set_copilot_view_setting
 
 
-class Completion:
+class ViewCompletionManager:
```

Comment:
What if we rename this to `ViewSuggestionManager`(together with `_PopupSuggestion`)?

Currently use `copilot_accept_suggestion`/`copilot_reject_suggestion` naming but under the hood we use `completion` work. I know that it doesn't really change anything, but I think it is important to stay consistent.

Alternatively, we could rename `copilot_accept_suggestion` to `copilot_accept_completion`(and `*_reject_` too) but I personally think that suggestion is more appropriate here as completion confuses with the Sublime completions

Reply:
I actually thought to do it in a reversed way to sync with the server protocol naming. https://github.com/TheSecEng/LSP-copilot/blob/8520ae9033d2c4a4ad117c227630cf172c763d02/constants.py#L11-L13

Reply:
I think both work, unless we stay consistent

Reply:
[refactor: rename "suggestion" to "completion"](https://github.com/TheSecEng/LSP-copilot/pull/13/commits/71be8312ea75ff5f24bedff44fc21a0e4aa691f3) 

@timfjord This change incudes command names so your custom keybindings have to be renamed as well.